x/oracle/keeper.go

```diff
@@ -282,8 +280,9 @@ func (k Keeper) DeleteAggregateExchangeRateVote(ctx sdk.Context, voter sdk.ValAd
 }
 
 // IterateAggregateExchangeRateVotes iterates rate over prevotes in the store
-func (k Keeper) IterateAggregateExchangeRateVotes(ctx sdk.Context, handler func(voterAddr sdk.ValAddress,
-	aggregateVote types.AggregateExchangeRateVote) (stop bool)) {
+func (k Keeper) IterateAggregateExchangeRateVotes(ctx sdk.Context,
+	handler func(voterAddr sdk.ValAddress,
+		aggregateVote types.AggregateExchangeRateVote) (stop bool)) {
```

Comment:
This is difficult to read. Perhaps consider making `func(voterAddr sdk.ValAddress,
		aggregateVote types.AggregateExchangeRateVote) (stop bool)` a type alias.